### DISCUSSION TITLE
Another font-type (sans-serife-based) for connected displays?

### DISCUSSION THREAD

**Discussion by LeChrisDE** [tbnobody/OpenDTU#1099] [General]
1) Is it possible to show a much better readable font-type for the connected Displays? My suggestion would be a sans-serife font-type like the OpenDTU web UI offers. In my picture you can see the display with a serife based font-type which is not really good readable.

2) If not directly, does anybody know another display-type, which offers another font?

3) Any other ideas?

![PXL_20230629_191805536 PORTRAIT](https://github.com/tbnobody/OpenDTU/assets/63072521/96dbda0c-6ad6-4458-91c9-8fea024810c6)

**Comment by Henrik-Ingenieur** [comment]
Hi LeChrisDe,

your can adjust the font easily by your own. Open `src/Display_Graphic.cpp` and find the function `DisplayGraphicClass::setFont`. There you can set the Font you like to have.
Which fonts you can choose from is given here: [U8g2->Fonts](https://github.com/olikraus/u8g2/wiki/fntlistall)

Happy coding.

**Reply by LeChrisDE** [reply]
Hello Henrik,

Thank you very much for your help and advice.

Best regards,
Chris
